html/editing/editing-0/autocorrection/autocorrection.html

```diff
@@ -0,0 +1,320 @@
+<!DOCTYPE html>
+<html>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/interaction.html#autocorrection">
+<body>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+
+test(function() {
+    assert_true('autocorrect' in document.createElement('input'));
+}, "Test that the autocorrect is available on HTMLInputElement.")
+
+test(function() {
+    assert_true('autocorrect' in document.createElement('textarea'));
+}, "Test that the autocorrect is available on HTMLTextAreaElement.")
+
+test(function() {
+    assert_true('autocorrect' in document.createElement('div'));
+}, "Test that the autocorrect is available on div.")
+
+test(function() {
+    assert_true('autocorrect' in document.createElement('form'));
+}, "Test that the autocorrect is available on form.")
+
+test(function() {
+  var elements = [ document.createElement('input'),
+                   document.createElement('textarea'),
+                   document.createElement('div'),
+                   document.createElement('form') ];
+
+  elements.forEach(function(e) {
+    e.autocorrect = true;
+    assert_true(e.autocorrect);
+    assert_equals(e.getAttribute('autocorrect'), 'on');
+
+    e.autocorrect = 'hello';
+    assert_true(e.autocorrect);
+    assert_equals(e.getAttribute('autocorrect'), 'on');
+
+    e.autocorrect = false;
+    assert_false(e.autocorrect);
+    assert_equals(e.getAttribute('autocorrect'), 'off');
+
+    e.autocorrect = 0;
+    assert_false(e.autocorrect);
+    assert_equals(e.getAttribute('autocorrect'), 'off');
+  });
+}, "Test setting the autocorrect boolean attribute.");
```

Comment:
Nit: you want "IDL attribute" here as boolean attribute is a term of art in HTML that means something else.